js/components/controlBar.js

```diff
@@ -259,8 +259,13 @@ var ControlBar = React.createClass({
         "oo-volume-bar": true,
         "oo-on": turnedOn
       });
-      var barStyle = { backgroundColor: this.props.skinConfig.controlBar.volumeControl.color ? this.props.skinConfig.controlBar.volumeControl.color : this.props.skinConfig.general.accentColor };
-
+      var barStyle = turnedOn === true ? ({
```

Comment:
Can you please change this to use if/else?  It's very difficult to follow and very likely we will mess something up if we have to edit it.

Reply:
Modified as you said.